feature/settings/src/main/kotlin/com/patrykandpatryk/liftapp/feature/settings/ui/Settings.kt

```diff
@@ -19,6 +29,10 @@ fun Settings(
     modifier: Modifier = Modifier,
 ) {
     val topAppBarScrollBehavior = topAppBarScrollBehavior()
+    val viewModel = hiltViewModel<SettingsViewModel>()
+
+    val massUnit by viewModel.massUnit.collectAsState(initial = null)
+    val distanceUnit by viewModel.distanceUnit.collectAsState(initial = null)
```

Comment:
Once we have more preferences, the number of local variables will grow. It would be better to use the same approach, as in `OneRepMax` screen — A UI state holding these values.

Reply:
Please see 27667158dbbb6aa4bb4401138968ebd596d5b204.